context/conversationStore.ts

```diff
@@ -0,0 +1,38 @@
+import { ConversationActionType, IConversationState } from './contextTypes';
+
+export default function conversationReducer(
+  state: IConversationState,
+  action: ConversationActionType
+): IConversationState {
+  switch (action.type) {
+    case 'SET_ALL_CONVERSATIONS':
+      console.log('SET_ALL_CONVERSATIONS');
```

Comment:
Did you mean to leave this log in here?

Reply:
You know we did't 🚫 👻  - good catch!

I removed it in a new commit